composer.json

```diff
@@ -16,6 +16,7 @@
     "require": {
         "php": "^7.2 || ^8.0",
         "doctrine/common": "^2.13|^3.0",
+        "doctrine/dbal": "^2.13|^3.0",
```

Comment:
Classes from the `doctrine/dbal` namespace are used directly in the `lib/Doctrine/Common/DataFixtures` so it should be in `require` and not `require-dev`.

Reply:
Are you sure? I can see `doctrine/mongodb-odm` in the dev dependencies. Should a MongoDB project really install `doctrine/dbal`?

Reply:
the DBAL classes are used only in the DBAL-specific code, which is optional in the library. So this should indeed be reverted (using a conflict rule to specify that the library is incomaptible with DBAL <2.13 instead)

Reply:
This line still needs to be reverted.

Reply:
Removed.